mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -1538,7 +1538,8 @@ def _compute_moc(latBins, nz, latCell, regionCellMask, transportZ,
     """compute meridionally integrated MOC streamfunction"""
 
     mocTop = np.zeros([np.size(latBins), nz + 1])
-    mocTop[0, range(1, nz + 1)] = transportZ.cumsum()
+    mocSouthBottomUp = - transportZ[::-1].cumsum()
+    mocTop[0, 0:nz] = mocSouthBottomUp[::-1]
```

Comment:
@vanroekel, the previous calculation didn't work as expected, I think because I had the indexing wrong (negative indices are tricky!).  But this seems to work.